packages/viewer/src/lib/scenes/scene.ts

```diff
@@ -236,6 +236,7 @@ export class Scene {
     private decodeFrame: FrameDecoder,
     private imageScaleProvider: ImageScaleProvider,
     private dimensions: Dimensions.Dimensions,
+    public readonly sceneId: UUID.UUID,
```

Comment:
Question: Why not a private variable with a public getSceneId function?

Reply:
That's equivalent to `public readonly`, but requires more code.